src/patternfly/components/Tabs/examples/tabs-secondary.hbs

```diff
@@ -0,0 +1,39 @@
+{{#> tabs tabs--attribute='aria-label="Primary tabs with secondary example"'}}
+  {{#> tabs-list}}
+    {{#> tabs-item tabs-item--current="true"}}
+      {{#> tabs-button tabs-button--attribute='id="ex3-tab1"'}}
+        Tab item 1
+      {{/tabs-button}}
+    {{/tabs-item}}
+    {{#> tabs-item}}
+      {{#> tabs-button tabs-button--attribute='id="ex3-tab2""'}}
+        Tab item 2
+      {{/tabs-button}}
+    {{/tabs-item}}
+    {{#> tabs-item}}
+      {{#> tabs-button tabs-button--attribute='id="ex3-tab3"'}}
+        Tab item 3
+      {{/tabs-button}}
+    {{/tabs-item}}
+  {{/tabs-list}}
```

Comment:
Might be easier to understand if you make the tab titles longer and/or add several more. It would help illustrate the scroll buttons appearing only when tabs reach the overflow point.

Reply:
This particular code chunk is for the secondary tabs example though, with scroll buttons off. I have 6 tab items in the scroll buttons example - do you think I should add more there?

Reply:
Yes, sorry. I put the comment on the wrong file.

Reply:
Done.